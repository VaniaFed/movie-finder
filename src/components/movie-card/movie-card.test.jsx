import { shallow } from 'enzyme';
import { v4 } from 'uuid';
import { MovieCard } from './index';

const movie = {
    title: 'movei1',
    genres: 'action',
    tagline: 'some tagline',
    releaseDate: '2014',
    posterPath:
        'https://pp.userapi.com/c844521/v844521991/f808b/TCjtAXN8SdA.jpg',
    id: v4()
};

describe('<MovieCard />', () => {
    const tree = shallow(<MovieCard movie={movie} />);
    it('should be rendered with movie', () => {
        expect(tree).toMatchSnapshot();
    });
});
