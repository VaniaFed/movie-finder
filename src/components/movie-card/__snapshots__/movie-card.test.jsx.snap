// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCard /> should be rendered with movie 1`] = `ShallowWrapper {}`;
